commands/lxc2lxd.go

```diff
@@ -60,3 +62,70 @@ cat << 'EOF' > /var/lib/juju/1.25-upgrade/scripts/lxc-to-lxd
 	}
 	return nil
 }
+
+// StopLXCContainer starts the specified LXC container machine.
+func StartLXCContainer(container, host description.Machine) error {
```

Comment:
Sorry, I originally had them all taking descriptions because the verify-source command had descriptions. I then found that that was a bit awkward for the remaining commands. StartLXCContainer isn't used yet. I'll just drop it for now.